docker/helloworld/main.go

```diff
@@ -62,13 +63,18 @@ func main() {
 		s := grpc.NewServer()
 		server, err := grpcServer.New(serverText, *target, podIp, *latency, *failureRate)
 		dieIf(err)
-		proto.RegisterSvcServer(s, server)
+
+		isHello, err := regexp.MatchString("(?i)hello", serverText)
+		if isHello {
```

Comment:
Tiny nit -- you're not checking the value of `err` here. I think it might be simpler to write this as:

```
if strings.ToLower(serverText) == "hello" {
	...
} else {
	...
}
```